phing/tasks/deploy.xml

```diff
@@ -84,6 +84,12 @@
     <phingcall target="target-hook:invoke">
       <property name="hook-name" value="post-deploy-build"/>
     </phingcall>
+    <if>
+      <isset property="simplesamlphp"/>
```

Comment:
Rather than relying on new variable, could we instead just verify that simple saml php is being used in the project via `composer info` or something similar?